src/components/Image/Image.tsx

```diff
@@ -60,6 +84,8 @@ function WithLink({
       {children}
     </ExtLink>
   ) : (
-    <div className={className}>{children}</div>
```

Comment:
Wrapping in `div` was causing `ImgCropper` to display a very small image on screen when using this component. Rendering `img` directly like this enables it (and by extension `ImgCropper`) to render a bigger image (up to [defined max size](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/img-to-image/src/components/ImgEditor/ImgCropper.tsx#L50)).